### ISSUE TITLE
After Flashing 2.31

### ISSUE THREAD

**Issue by sandog78** [ClusterM/hakchi2#1390]
I am trying to flash a new nes classic and after flashing when I power up it shows enter password on tv screen. Any Ideas whats wrong? Here is a link to show issue

https://1drv.ms/u/s!Ar_TTAR2HGYxzXVALKZQw1ve4HGv

**Comment by Brds7t7** [comment]
Go to modules menu -> Remove modules -> Remove the Password.hmod

**Comment by sandog78** [comment]
ok.. What ver of hackchi is the best for the latest release of the nes classic. Also should I flash before removing removing password modules.

**Comment by Brds7t7** [comment]
Use whichever version suits you. Some people use CE, I couldn't get it to do anything though. I tried all the v1.2.x versions and it wouldn't get past the rebooting stage when I tried to do anything with it, so I went back to Cluster's 2.21f. That version works fine for me on 2 older SNES minis and 2 newer NES minis. I get the MD5 mismatch error, but it hasn't caused me any issues with adding, deleting games, flashing or dumping kernel, etc. I use it with KMFDManic's coreset. 

Haven't used the newer versions because I'm not interested in using USB-HOST. I use a raspberry pi for most of my other games.

No need to flash anything before. Just plug the SNES mini into PC, open Hakchi and select to remove the password.hmod, the onscreen instructions will tell you what to do next.

Remember to close this issue if it works.
